tests/dummy/app/controllers/application.js

```diff
@@ -218,6 +219,10 @@ export default class ApplicationController extends Controller {
     this.mockUploader = owner.lookup('service:mock-uploader');
   }
 
+  @action onRatingClick(rating) {
```

Comment:
```suggestion
  @action
  onRatingClick(rating) {
```
Set `onRatingClick` on a new line